tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es2019",
-    "lib": ["es2019"],
+    "target": "es2021",
```

Comment:
Out of scope of this PR but take a look at presets like https://www.npmjs.com/package/@tsconfig/node16 that set some of these options for the target Node version.